stix2generator/interop_custom.json

```
@@ -0,0 +1,144 @@
+{
+    "attack-pattern": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["external_references", "kill_chain_phases"],
+        "properties":{
+            "external_references":{
+                "type": "array",
+                "ref": "external-references"
+            }
+        }
+    },
+    "campaign": {
+        "type": "object",
+        "import": "common-properties"
+    },
+    "course-of-action": {
+        "type": "object",
+        "import": "common-properties"
+    },
+    "grouping": {
+        "type": "object",
+        "import": "common-properties"
+    },
+    "incident": {
+        "type": "object",
+        "import": "common-properties"
+    },
+    "identity": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["identity_class"]
+    },
+    "indicator": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["indicator_types","name"]
+    },
+    "infrastructure": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["infrastructure_types"]
+    },
+    "intrusion-set": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["resource_level", "primary_motivation"]
+    },
+    "location": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["region"]
+    },
+    "malware": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["malware_types", "first_seen", "last_seen", "architecture_execution_envs", "implementation_languages", "capabilities"]
```

These properties are still required for interop:
```suggestion
        "required": ["is_family", "name", "malware_types", "first_seen", "last_seen", "architecture_execution_envs", "implementation_languages", "capabilities"]
```